client/js/components/home/assessmentCreationForm.js

```diff
@@ -6,12 +6,21 @@ var TemplateSelect = require('./templateSelect')
 var $ = require('jquery')
 
 var AssessmentCreationForm = React.createClass({
+  propTypes: {
+    initialTags: React.PropTypes.array,
+    team: React.PropTypes.object
+  },
   getInitialState: function () {
     return {
       template: '',
       tags: ''
     }
   },
+  componentDidMount: function () {
+    this.setState({
+      tags: this.props.initialTags
+    })
+  },
```

Comment:
> I can't figure out how to default the values of the tags to the team tags for a team assessment. Any ideas?
> 
> https://measuremate-pr-124.herokuapp.com/
> 
> add-team-support branch. 

The *initialTags* values as asynchronously loaded by a parent component ajax call.  The initial time the child component is loaded, initialTags property will be empty.

componentDidMount only fires once when a component is initially created. To accommodate additional moments when a component receives new properties from its parent you need to implement a *willReceiveProps* method on your component

Something like this could do the trick:

```
  componentWillReceiveProps: function (nextProps) {
    this.setState({
      tags: nextProps.initialTags
    })
  },
```

I frequently go back to this reference:
https://facebook.github.io/react/docs/component-specs.html